app/containers/TrackDetailsContainer/index.js

```diff
@@ -0,0 +1,84 @@
+/**
+ *
+ * TrackDetailsContainer
+ *
+ */
+
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { connect } from 'react-redux';
+import { compose } from 'redux';
+import { Card, Typography, Space, Avatar } from 'antd';
+import { musicContainerCreators } from '../MusicContainer/reducer';
```

Comment:
TrackDetails should have its own container. It should not import from a sibling container.

Reply:
How do you access another state from another reducer?

Reply:
you shouldn't